modules/nw-ingress-controller-status.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-status-{context}"]
+= View Ingress controller status
+
+Your can view the status of a particular Ingress controller.
```

Comment:
The capitalization is odd—is there a reason to capitalize "Ingress" here?

Reply:
No, probably just got in the habit since we're capitalizing it for the operator. Fixing, thanks!